source/elements/oneMKL/source/domains/data_fitting/interpolate.rst

```diff
@@ -0,0 +1,106 @@
+.. _interpolate:
+
+Interpolate Function
+====================
+
+.. contents::
+    :local:
+    :depth: 1
+
+Interpolate function performs computations of function and derivatives values at interpolation sites.
+
+If the sites do not belong to interpolation interval ``[a, b]``, the library uses:
+
+  - interpolant :math:`I_0` coefficients computed for interval :math:`[x_0, x_1)` for the
+    computations at the sites to the left of ``a``.
+  - interpolant :math:`I_{n-2}` coefficients computed for interval
+    :math:`[x_{n-2}, x_{n-1})` for the computations at the sites to the right of ``b``.
```

Comment:
You mean to say that you do allow for extrapolation, but it uses the interpolant function defined on the closest sub-interval in the interpolation interval :math:`[a,b)`?

Reply:
This seems different than your definition of site in the terms.rst

Reply:
> You mean to say that you do allow for extrapolation, but it uses the interpolant function defined on the closest sub-interval in the interpolation interval :math:`[a,b)`?

Correct.

> This seems different than your definition of site in the terms.rst

Statements here are correct. Will rework "terms.rst"